CHANGELOG.md

```diff
@@ -18,7 +18,7 @@
   - Fixed typos in [README.md](./README.md)
 
 ## 9.0.0
-##### 31 october 2021
+##### 31 October 2021
```

Comment:
Hello,

The month names in lowercase are meant to stay like that because the changelog has been always used like that and is fetched on some of my API, so I don't want to patch them for now :)

Reply:
ok